fuels-core/src/lib.rs

```diff
@@ -1,10 +1,13 @@
+//! This can be compiled with `#[no_std]`, ready for Wasm.
+#![cfg_attr(not(feature = "std"), no_std)]
+
 use core::fmt;
 use fuel_types::bytes::padded_len;
 use fuel_types::Word;
 use strum_macros::EnumString;
 
 pub mod abi_decoder;
-#[cfg(not(feature = "no-std"))]
+#[cfg(any(feature = "std"))]
```

Comment:
Is there a reason not to use

```suggestion
#[cfg(feature = "std")]
```

Reply:
No special reason.